relayer/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/btcsuite/btcd v0.20.1-beta // indirect
 	github.com/centrifuge/go-substrate-rpc-client v2.0.0-rc6+incompatible
 	github.com/deckarep/golang-set v1.7.1 // indirect
-	github.com/ethereum/go-ethereum v1.9.22
+	github.com/ethereum/go-ethereum v1.8.23
```

Comment:
This is the highest version that ethashproof builds with :( I didn't want to spend time upgrading ethashproof but happy to create an issue and do this later if you think it's important

Reply:
Since this should be highly security relevant, I feel we need to upgrade ethashproof before going to production. See https://github.com/ethereum/go-ethereum/releases for details on the security issues fixed in previous releases.

Reply:
Hm I don't think ethash has changed at all since then but it would still be a good idea to upgrade yeh. we can add a github issue for this to address later.

Reply:
https://github.com/Snowfork/polkadot-ethereum/issues/169